### DISCUSSION TITLE
bookmark in google

### DISCUSSION THREAD

**Discussion by tienle2003** [dailydotdev/daily#1047] [General]
before this time, when I used Google Chrome bookmarks also showed on it, bookmarks were very convenient to help me easily choose and switch between different websites but today bookmarks are hidden when I use your service so very inconvenient for me. So can you recover it?

**Comment by nimrodkra** [comment]
It’s a browser setting actually. Here’s a quick guide to help you fix
that:
https://docs.daily.dev/docs/getting-started/browser-extension-installation#my-chromes-bookmarks-bar-is-gone-how-can-i-bring-it-back
